common/tables/table-en.html

```diff
@@ -0,0 +1,520 @@
+---
+{
+	"title": "GCWeb Table example",
+	"language": "en",
+	"altLangPage": "table-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-02-28"
+}
+---
+
+<h2 id="tables-example" class="wb-prettify">Basic example</h2>
+<p>For basic styling&mdash;light padding and only horizontal dividers&mdash;add the base class <code>.table</code> to any <code>&lt;table&gt;</code>. It may seem super redundant, but given the widespread use of tables for other plugins like calendars and date pickers, we've opted to isolate our custom table styles.</p>
+<div class="bs-example" data-example-id="simple-table">
+	<table class="table">
+		<caption>Optional table caption.</caption>
+		<thead>
+			<tr>
+				<th>#</th>
```

Comment:
It will be preferable to use word instead of a symbol.

```suggestion
				<th>Number</th>
```